build/build-parent/pom.xml

```diff
@@ -329,17 +334,6 @@
             <newArtifacts>
               <artifact>${project.groupId}:${project.artifactId}:${project.version}</artifact>
             </newArtifacts>
-            <!-- By default revapi will check the oldArtifact against the currently executed build -->
-            <analysisConfigurationFiles>
-              <configurationFile>
-                <path>src/build/revapi-config.json</path>
-                <roots>
-                  <root>filters</root>
-                  <root>ignores</root>
-                </roots>
-              </configurationFile>
-            </analysisConfigurationFiles>
-            <failSeverity>potentiallyBreaking</failSeverity>
```

Comment:
Is this configuration no longer relevant?

Reply:
The configuration, turns out, is the default.